### PR TITLE
* #5: Changed the JSON response format 

### LINKED ISSUE #5: Improve the response JSON format
## Problem

We need to define the JSON format for responses. We could use the same format as WebSockets or we can modify it slightly.
Right now we return the WebSockets JSON directly on 200 code and some custom error code + error message on 400.

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #5

### CODE REVIEW

**Review by pj-spoelders — DISMISSED** [review]
Looks good! Thanks Nastya!

**Review by pj-spoelders — APPROVED** [review]
ok

File: doc/design.md [hunk 1]
```diff
@@ -339,7 +371,42 @@ Needs: impl, itest
...
+        "name": <string>,
+        "dataType": {
+          "type": <string>,
+          "precision": <number>,
+          "scale": <number>,
+          "size": <number>,
+          "characterSet": <string>,
+          "withLocalTimeZone": <true
+          |
+          false>,
```
Comment (kaklakariada) [inline doc/design.md:399]:
```suggestion
          "withLocalTimeZone": <true | false>,
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]
Minor finding

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**